PhysicsTools/ONNXRuntime/interface/ONNXRuntime.h

```diff
@@ -22,13 +22,20 @@ namespace cms::Ort {
 
   typedef std::vector<std::vector<float>> FloatArrays;
 
+  enum class Backend {
+    cpu,
+    cuda,
```

Comment:
I think the `cuda` should be removed to avoid confusion (there is no CUDA support for ONNX in 10_6_X).

Alternatively the whole `Backend` enum is unnecessary.

Reply:
I removed ```cuda```